.github/comment.yml

```diff
@@ -1,20 +1,46 @@
+    onLabeledComments: >-
+      Hi @${PRAuthor}! The automation detected breaking changes in this pull request.
+      As a result, it added the `BreakingChangeReviewRequired` label.
+      <br/><br/>
+      You cannot proceed with merging this PR until you complete one of the following action items:
+      <br/><br/>
+      **ACTION ITEM A**: To get your breaking changes approved,
+      please follow the process described in the
+      [High-level Breaking Change Process doc](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-core-pm-and-design/trusted-platform-pm-karimb/service-lifecycle-and-actions-team/service-lifecycle-actions-team/apex/media/overview_breakingchanges#high-level-breaking-change-process).
+      <br/><br/>
+      **ACTION ITEM B**:
+      Alternatively, if you think the breaking changes are false positives 
+      (false alarms), please explain why in a PR comment and @ the PR assignee.
+      <br/><br/>
+      If you want to know the production traffic statistics,
+      please see the [ARM Traffic statistics](${openapiHub}/api/pr/query-traffic?pr=${owner}/${repo}/${PRNumber}&days=5).
+
+- rule:
+    type: label
+    label: NewApiVersionRequired
+    booleanFilterExpression:
+    onLabeledComments: >-
+      Hi @${PRAuthor}! The automation detected this pull request introduces breaking changes 
+      to an existing API version that can be avoided by creating a new API version instead.
+      As a result, the automation added the `NewApiVersionRequired` label.
+      <br/><br/> 
+      You cannot proceed with merging this PR until you complete one of the following action items:
+      <br/><br/>
+      **ACTION ITEM A**: Submit a new PR instead of this one, or modify this PR, 
+      so that it adds a new API version instead of introducing breaking changes to an existing API version. 
```

Comment:
```suggestion
      so that it adds a new API version instead of introducing changes to an existing API version. 
```

Reply:
@mikekistler I changed this but then I see small conceptual inconsistency. That is, if we have label `BreakingChangeReviewRequired` the PR author can request approval for breaking changes. But if we have `NewApiVersionRequired` then the PR author can request approval for changes. Just changes, not necessarily breaking. But they still need to consult a doc that talks about _breaking_ changes. Which is a bit inconsistent possibly a bit confusing.

Am I correct in my understanding on what is going on?

Reply:
Your understanding is correct and I agree that is is not what we want it to be. We should really have a different label to approve "NoApiVersionChange".

Reply:
Thx! Filed an issue:
- https://github.com/Azure/azure-sdk-tools/issues/6344